scanners/ios/podfile.go

```diff
@@ -81,15 +92,29 @@ end
 		return map[string]string{}, fmt.Errorf("failed to parse target definition output, error: %s", err)
 	}
 
-	if targetDefinitionOutput.Error != "" {
+	if podfileParser.shouldRaiseReadDefinitionError(targetDefinitionOutput.Error) {
 		return map[string]string{}, fmt.Errorf("failed to read target defintion map, error: %s", targetDefinitionOutput.Error)
 	}
 
 	return targetDefinitionOutput.Data, nil
 }
 
-func getUserDefinedProjectRelavtivePath(podfilePth, cocoapodsVersion string) (string, error) {
-	targetProjectMap, err := getTargetDefinitionProjectMap(podfilePth, cocoapodsVersion)
+func (podfileParser podfileParser) shouldRaiseReadDefinitionError(err string) bool {
+	if err == "" {
+		return false
+	}
+
+	isInvalidPodfileError := stringutil.CaseInsensitiveContains(err, "Pod::DSLError")
```

Comment:
Why is the case insensitive match needed?

Reply:
You are right, it is no longer needed. I was comparing different strings for which it made sense.